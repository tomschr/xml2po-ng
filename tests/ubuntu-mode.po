msgid ""
msgstr ""
"Project-Id-Version: превод\n"
"POT-Creation-Date: 2006-04-01 18:46+0200\n"
"PO-Revision-Date: 2006-04-01 18:47+0200\n"
"Last-Translator: Данило\n"
"Language-Team: српски\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: website-index.xml:11(title)
msgid "Ubuntu Documentation"
msgstr "Убунту документација"

#: website-index.xml:12(para)
msgid ""
"Welcome to the Ubuntu Documentation Website, for <emphasis role=\"bold"
"\">Ubuntu &distro-rev;</emphasis>, <emphasis>&distro-version;</emphasis>!"
msgstr ""

#: website-index.xml:13(para)
msgid ""
"This site is where you can find the official documentation developed and "
"maintained by the Ubuntu Documentation Project. The following guides are "
"available:"
msgstr ""

#: website-index.xml:16(para)
msgid ""
"<emphasis role=\"bold\">Ubuntu Desktop Guide</emphasis> - <ulink url="
"\"ubuntu/desktopguide/&language;/index.html\">HTML</ulink>, <ulink url=\"pdf/"
"ubuntu/&language;/desktopguide.pdf\">PDF</ulink>"
msgstr "<emphasis role=\"bold\">Упутство за Убунту окружење</emphasis> — <ulink url=\"ubuntu/desktopguide/&language;/index.html\">HTML</ulink>, <ulink url=\"pdf/ubuntu/&language;/desktopguide.pdf\">PDF</ulink>"

#: website-index.xml:19(para)
msgid ""
"<emphasis role=\"bold\">Kubuntu Desktop Guide</emphasis> - <ulink url="
"\"kubuntu/desktopguide/&language;/index.html\">HTML</ulink>, <ulink url="
"\"pdf/kubuntu/&language;/desktopguide.pdf\">PDF</ulink>"
msgstr ""

#: website-index.xml:22(para)
msgid ""
"<emphasis role=\"bold\">Server Guide</emphasis> - <ulink url=\"ubuntu/"
"serverguide/&language;/index.html\">HTML</ulink>, <ulink url=\"pdf/ubuntu/"
"&language;/serverguide.pdf\">PDF</ulink>"
msgstr ""

#: website-index.xml:25(para)
msgid ""
"<emphasis role=\"bold\">Packaging Guide</emphasis> - <ulink url=\"ubuntu/"
"packagingguide/&language;/index.html\">HTML</ulink>, <ulink url=\"pdf/ubuntu/"
"&language;/packagingguide.pdf\">PDF</ulink>"
msgstr ""

#: website-index.xml:28(para)
msgid "Find out more about Ubuntu and Kubuntu from the following documents:"
msgstr ""

#: website-index.xml:31(para)
msgid ""
"<emphasis role=\"bold\">Ubuntu</emphasis> - <ulink url=\"ubuntu/about-ubuntu/"
"&language;/index.html\">About Ubuntu</ulink>, <ulink url=\"ubuntu/release-"
"notes/&language;/index.html\">Ubuntu Release Notes</ulink>"
msgstr ""

#: website-index.xml:34(para)
msgid ""
"<emphasis role=\"bold\">Kubuntu</emphasis> - <ulink url=\"kubuntu/about-"
"kubuntu/&language;/index.html\">About Kubuntu</ulink>, <ulink url=\"kubuntu/"
"release-notes/&language;/index.html\">Kubuntu Release Notes</ulink>"
msgstr ""

#: website-index.xml:37(para)
msgid ""
"If you can't find what you are looking for here, try the excellent source of "
"community contributed documentation on the <ulink url=\"https://wiki.ubuntu."
"com/UserDocumentation\">Ubuntu wiki</ulink>."
msgstr ""

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: website-index.xml:0(None)
msgid "translator-credits"
msgstr ""
