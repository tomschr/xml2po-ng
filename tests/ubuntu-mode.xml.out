<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE article PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN" "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" [
<!ENTITY % globalent SYSTEM "../../libs/global.ent">
<!ENTITY % cdo-C SYSTEM "../../libs/cdo-C.ent">
<!ENTITY language "sr">
]>
<article lang="sr">
	<title>Убунту документација</title>
		<para>Welcome to the Ubuntu Documentation Website, for <emphasis role="bold">Ubuntu &distro-rev;</emphasis>, <emphasis>&distro-version;</emphasis>!</para>
		<para>This site is where you can find the official documentation developed and maintained by the Ubuntu Documentation Project. The following guides are available:</para>
		<itemizedlist>
			<listitem>
				<para><emphasis role="bold">Упутство за Убунту окружење</emphasis> — <ulink url="ubuntu/desktopguide/&language;/index.html">HTML</ulink>, <ulink url="pdf/ubuntu/&language;/desktopguide.pdf">PDF</ulink></para>
			</listitem>
			<listitem>
				<para><emphasis role="bold">Kubuntu Desktop Guide</emphasis> - <ulink url="kubuntu/desktopguide/&language;/index.html">HTML</ulink>, <ulink url="pdf/kubuntu/&language;/desktopguide.pdf">PDF</ulink></para>
			</listitem>
			<listitem>
				<para><emphasis role="bold">Server Guide</emphasis> - <ulink url="ubuntu/serverguide/&language;/index.html">HTML</ulink>, <ulink url="pdf/ubuntu/&language;/serverguide.pdf">PDF</ulink></para>
			</listitem>
			<listitem>
				<para><emphasis role="bold">Packaging Guide</emphasis> - <ulink url="ubuntu/packagingguide/&language;/index.html">HTML</ulink>, <ulink url="pdf/ubuntu/&language;/packagingguide.pdf">PDF</ulink></para>
			</listitem>
		</itemizedlist>
		<para>Find out more about Ubuntu and Kubuntu from the following documents:</para>
		<itemizedlist>
			<listitem>
				<para><emphasis role="bold">Ubuntu</emphasis> - <ulink url="ubuntu/about-ubuntu/&language;/index.html">About Ubuntu</ulink>, <ulink url="ubuntu/release-notes/&language;/index.html">Ubuntu Release Notes</ulink></para>
			</listitem>
			<listitem>
				<para><emphasis role="bold">Kubuntu</emphasis> - <ulink url="kubuntu/about-kubuntu/&language;/index.html">About Kubuntu</ulink>, <ulink url="kubuntu/release-notes/&language;/index.html">Kubuntu Release Notes</ulink></para>
			</listitem>
		</itemizedlist>
		<para>If you can't find what you are looking for here, try the excellent source of community contributed documentation on the <ulink url="https://wiki.ubuntu.com/UserDocumentation">Ubuntu wiki</ulink>.</para>
</article>
